tcp/server.go

```diff
@@ -1,10 +1,99 @@
 package qbinTCP
 
 import (
+	"io/ioutil"
+	"net"
+	"strings"
+	"time"
+
 	"github.com/qbin-io/backend"
 )
 
 // StartTCP launches the TCP server which is responsible for the TCP API.
+//listen <- given tcp port
 func StartTCP(listen string, root string) {
-	qbin.Log.Warning("TCP server is not yet implemented.")
+	root = strings.TrimSuffix(root, "/")
+
+	//reading server TCP-Address
+	tcpAddr, err := net.ResolveTCPAddr("tcp", listen)
+	checkErr(err)
```

Comment:
As different errors in different places will be handled very differently here (e.g. here it's only logged and returns, in incoming connections it's only logged and continues, in handleClient it's only sent back to the client, and so on) it might be possible to omit the `checkErr()` function completely.